src/frontend/src/views/Data/components/SamplesView/index.tsx

```diff
@@ -27,6 +28,15 @@ const SamplesView = (): JSX.Element => {
 
   // load sample data from server
   const { data: samples, isLoading } = useSampleInfo();
+  const [showBlankState, setShowBlankState] = useState<boolean>(false);
+
+  useEffect(() => {
```

Comment:
I don't think the useEffect is needed here. Couldn't think be a regular if statement that returns a blank state if no samples are available?

Reply:
Was thinking that we would want to update this if someone deleted their last sample.  Would the component re-render in that case?